clash-lib/src/Clash/Driver/Manifest.hs

```diff
@@ -65,7 +72,58 @@ data Manifest
   , fileNames :: [(FilePath, ByteString)]
     -- ^ Names and hashes of all the generated files for the @TopEntity@. Hashes
     -- are SHA256.
-  } deriving (Show,Read)
+  } deriving (Show,Read,Eq)
+
+instance ToJSON Manifest where
+  toJSON (Manifest{..}) =
+    Aeson.object
+      [ "version" .= ("unstable" :: Text)
+      , "hash" .= manifestHash
```

Comment:
I wonder if this should also be a sha256 field / hash, if only for uniformity?

Reply:
Sure thing